tests/lib/ocpp/v201/test_composite_schedule.cpp

```diff
@@ -622,4 +622,62 @@ TEST_F(ChargepointTestFixtureV201, K08_CalculateCompositeSchedule_MaxOverridenBy
     ASSERT_EQ(actual, expected);
 }
 
+TEST_F(ChargepointTestFixtureV201, K08_CalculateCompositeSchedule_ExternalOverridesHigherLimits) {
```

Comment:
Reading these two tests, I am wondering if there isn't a way to communicate more clearly what these tests are doing.

In one case, I believe it is grabbing the default then getting something that overrides, and in the other it is the other way around? Is there ASCII art or something to make it clearer how to think through it? Otherwise, from here I can see the start and end times are different and that it leads to a specific expected schedule without really understanding why it should be like that. I would have to look at the JSON files, and study them, and perhaps I can be led to the conclusion better with an explanation here in the tests?

Reply:
This is probably something the whole test suite needs. The idea, as I currently understand it, is that your test make up a grid that covers every hour. The other profiles can then be added to either override or be overridden to assert against. The README in the json folders outline this. I think per-test an ASCII art would get messy so maybe at the top of the file? This is something I can work on with @folkengine.

This did make me realize I did not make updates to the external md.